src/utils/releases.js

```diff
@@ -48,7 +63,7 @@ async function generateReleaseNotes(token, newVersion, baseVersion) {
 
     logInfo(`Release notes generated: [${baseVersion} -> ${newVersion}]`)
 
-    return releaseNotes
+    return excludeUnwantedReleaseNotes(releaseNotes)
```

Comment:
maybe will be more readable if we assign the return value of `excludeUnwantedReleaseNotes()` to a variable and then return the variable.?